### PR TITLE
Page Details View: Show parent only if there is one

### LINKED ISSUE #51509: Page Details View: Show parent only if there is one
Instead of having Parent: Top-level (which could be confusing), I suggest not showing the parent item, if there is no parent (it's top-level). If this panel was editable then it makes sense to include, but otherwise not.

CleanShot 2023-06-14 at 16 06 26

### PR DESCRIPTION
**Pull request by juanfra**
## What?

Show the parent page in the page details view, only if there is one.

Fixes https://github.com/WordPress/gutenberg/issues/51509

## Why?

As reported in https://github.com/WordPress/gutenberg/issues/51509, having the parent details (set to top-level) when the page doesn't have a parent can be confusing.

## Testing Instructions

1. Create two pages, one without a parent and one with one.
2. Go to Appearance > Editor > Pages
3. Click on the page that has a parent, and confirm that the information is being displayed.
4. Go back, click on the page without a parent, and confirm that the "Parent" information is no longer there.

## Screenshots or screencast 

https://github.com/WordPress/gutenberg/assets/252415/11a79682-6fc4-4449-9904-7a1e805941b5

### CODE REVIEW

**Review by carolinan — APPROVED** [review]
The code change looks good to me.
The text shows correctly for pages with parents; and does not show for top-level pages.

File: packages/edit-site/src/components/sidebar-navigation-screen-page/page-details.js
Comment (Mamaduka) [inline packages/edit-site/src/components/sidebar-navigation-screen-page/page-details.js]:
```suggestion
			value: decodeEntities( page.parentTitle || __( '(no title)' ) ),
```

Nit: We already checked for the `undefined` above.

Comment (juanfra) [reply packages/edit-site/src/components/sidebar-navigation-screen-page/page-details.js]: Good catch! Thanks George :)

**Review by Mamaduka — APPROVED** [review]
Thank you, @juanfra! The changes here look good 👍

**Comment by jameskoster** [comment]
I think this makes sense while the detail is read only. But it's worth noting that we'll need to display it for top level pages too if we ever want to make it editable here.

**Comment by juanfra** [comment]
Thanks for the reviews ✨ 

@jameskoster is there a ticket with that initiative so that I can add these comments? For the time being, I'll merge the changes if you all agree.

**Comment by jameskoster** [comment]
It's something that has been discussed in design but there's no definitive direction yet. Certainly it's not something that should hold this PR up. I just wanted to leave a paper trail of the idea :)

**Comment by ramonjd** [comment]
Nice work. Thank you!

**Pull request merged**
